packages/css/src/components/anchor/anchor.template.ts

```diff
@@ -0,0 +1,54 @@
+import { Anchor } from '@dso-toolkit/sources';
+import { html, nothing } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined';
+
+import { iconTemplate } from '../icon/icon.template';
+
+export function anchorTemplate({ icon, iconAfter, iconOnly, label, modifier, url }: Anchor) {
+  return html`
+    <a href=${url}
+       title=${ifDefined(iconOnly ? label : undefined)}
+       class=${ifDefined(modifier)}
+       target=${ifDefined(modifier && modifier.indexOf('extern') > -1 ? '_blank' : undefined)}
+       rel=${ifDefined(modifier && modifier.indexOf('extern') > -1 ? 'noopener noreferrer' : undefined)}
+    >
```

Comment:
De gegenereerde HTML is anders. In de Nunjuck templates hadden we met `-` in expressies `{%- -%}` controle op whitespace. Lit kent dat niet, ik denk dat je de `${` opening naast `>` moet zetten

Reply:
Ik heb alle `${` openingen op elkaar geplakt, en zelfs dan is de gegenereerde HTML anders dan wat er in Fractal leeft. Moeten we dit niet gewoon als situatie accepteren? Hoe gaat dit straks in een WebComponent/React/Angular/etc. of zelfs bij een implementator die het op zijn manier in elkaar zet?